### DISCUSSION TITLE
Adding new artist results in empty group

### DISCUSSION THREAD

**Discussion by tkoreaper** [RandomNinjaAtk/docker-lidarr-extended#183] [Q&A] [answered]
I have attempted to add a few different artists and each one is empty. There are no albums/singles/etc.

**Comment by Yosh82** [comment, marked as answer]
Do you have some examples of the artists and I can test at my end. There are some, lets call them, uncommon artists, which I've seen without any releases, but assuming they list the releases on Musicbrainz (https://musicbrainz.org/), it should show the releases related to the artists.

**Reply by tkoreaper** [reply]
They're well known artists: The Offspring and The Smashing Pumpkins. I even tried using their musicbrainz id and they are still empty. I'm thinking that something may be broken on my end, but I don't know what could be preventing it from grabbing all the releases; it searches for them just fine.

**Reply by Yosh82** [reply]
So It adds the artists, but then shows no releases? If that's the case, check your profiles and ensure you've picked which type of releases you want:
![image](https://user-images.githubusercontent.com/106485860/211931677-97779bd7-4a4e-40ff-8522-3d351d29aee3.png)

Then, check the Artist Metadata Profile and ensure you've selected the one you want to use:
![image](https://user-images.githubusercontent.com/106485860/211931914-c4ce0400-4c68-48e9-aae5-31b87b720353.png)

So in my case, I only wanted their Studio Albums releases, and my profile for that is named "Default-Standard".

Hit refresh after and it should match your profile and find the relevant releases. Hope that helps.

**Reply by Yosh82** [reply]
It's worth noting that this isn't part of Lidarr-Extended, so you may want to check Lidaar's documentation which will explain this in more detail.

**Reply by Yosh82** [reply]
Once you've checked the above, if you're still having problems, post some snippets of your Profiles and Artists Metadata Profiles, and I'll see if I can help further. Good luck!

**Reply by tkoreaper** [reply]
Everything is set to the default that comes with Lidarr-extended... 'Any' quality profile with 'Standard' metadata profile. I never touched those settings.

I decided to reinstall and point it to a different named folder for its config to start fresh. (I kept the other config folder to go back to it if need be) After a while is detects all the artists I have in my music folder and I'm able to add new artists and get it to properly show all the releases, but for some reason it doesn't detect what music I have already even though it detected/added the artists. I do a full scan and it still shows I have nothing in spite of the music still being there. 
![image](https://user-images.githubusercontent.com/122491034/211942559-a57d87ca-a075-473c-aa4f-5ca8d4086bec.png)

I checked the permissions of my folders/files and I don't see any reason why it would be doing what it's doing.
![image](https://user-images.githubusercontent.com/122491034/211942978-99ed9516-84fe-4092-980b-e215321926b1.png)
![image](https://user-images.githubusercontent.com/122491034/211943048-b88b8ec2-fe42-47cc-8890-7165fbb09e65.png)

**Reply by Yosh82** [reply]
Check the docker logs to see what is doing. If you have fingerprint ID enabled in the settings etc, it can take a while to scan everything, the logs would be the best place to start  now it's showing the releases.

You could also try to manually import one artist directory and check the logs while it's doing that, but it should just auto import, assuming the paths are all correct.

**Reply by RandomNinjaAtk** [reply]
This seems like a problem with Lidarr itself... You may need to check with Lidarr support channels. I know lidarr has a difficult time with scanning/importing existing content.... I think Lidarr works best when it doesn't have pre-existing content at all and it acquires everything... But this is just based on personal experience...

**Reply by Yosh82** [reply]
I remember having a similar issue when I moved over to Lidar-Extended (as I had around 3TB of pre-existing music), and as I was having problems, I thought I would delete them all from Lidarr, and let it rescan. However, from memory, it added all my existing music to the "Import List Exclusions" asI deleted them from Lidarr, so when I rescanned, as they were in the excluded list, it didn't add them for this reason, so worth checking your exclusions list is empty (or clear it if it has listings in it) and try another scan.

**Reply by tkoreaper** [reply]
It appears to be working for now. I let it do its thing all night and it detected all of my music. Something in the previous config must have gotten borked somehow.
